_posts/en/newsletters/2022-07-06-newsletter.md

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Bitcoin Optech Newsletter #207'
+permalink: /en/newsletters/2022/07/06/
+name: 2022-07-06-newsletter
+slug: 2022-07-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Long-term block reward funding:** in a thread on the Bitcoin-Dev
+  mailing list ostensibly about [covenants][topic covenants], it was
+  noted that Bitcoin's long term security currently depends on
+  demand for block space.   That demand must generate transaction
+  fees to pay for Proof of Work (PoW) in excess of what an attacker
+  would be willing to purchase to disrupt Bitcoin users.  Developer
+  Peter Todd [pointed out][todd subsidy] that this dependency could be
+  removed if the Bitcoin protocol were modified to include a perpetual
+  subsidy.  Several respondents indicated that they thought the system
+  is better off without a perpetual subsidy, while others looked for
+  alternatives or apparent equivalencies such as demurrage.
+
+    As of this writing, the thread appears to consist of casual
+    conversation rather than advocacy for any particular proposal to
+    change Bitcoin in the near future.
+
+- **Updated alternative to BIP47 reusable payment codes:** developer
+  Alfred Hodler [posted][hodler new codes] to the Bitcoin-Dev mailing
+  list a proposal for an alternative to [BIP47][] that attempts to
+  address some of the issues found during its use in production.  BIP47
+  allows Alice to publish a payment code that anyone can use in
+  combination with their own keys to create private addresses for Alice
+  that only they and Alice will know, allowing anyone to reuse the
+  public payment code without engaging in privacy-harming [address
+  reuse][topic output linking].  One of the problems with BIP47 is that
+  the first transaction between a spender (e.g. Bob) and Alice may leak
+  to block chain analysis organizations that Bob wants to pay Alice.
+
+    Hodler's scheme would be less likely to leak this information but it
+    would increase the amount of data a client implementing the protocol
+    would need to learn from the block chain, making it less suitable
+    for light clients.  Ruben Somsen suggested several alternatives
```

Comment:
I read this at first as if it suggested that Ruben invented all of the following and stumbled on the comma before Robin Linus. I only grokked what I understand to be the intended meaning on the third glance. Maybe:

```suggestion
    for light clients.  Ruben Somsen indicated several alternatives
```